src/nls/hr/strings.js

```diff
@@ -62,7 +62,7 @@ define({
     "ERROR_CREATING_FILE"               : "Došlo je do greške prilikom pokušaja da se kreira {0} <span class='dialog-filename'>{1}</span>. {2}",
 
     // Application preferences corrupt error strings
-    "ERROR_PREFS_CORRUPT_TITLE"         : "Greška kod čitanja preferencija",
+    "ERROR_PREFS_CORRUPT_TITLE"         : "Greška prilikom učitanja preferencija",
```

Comment:
@diomed This Brackets README page describes how to make updates to branches created directly in browser: https://github.com/adobe/brackets/tree/master/src/nls#updating-existing-branch-and-pull-request